lib/puppet/info_service/task_information_service.rb

```diff
@@ -28,7 +28,7 @@ def self.task_data(environment_name, module_name, task_name)
 
     begin
       task.validate
-      {:metadata => task.metadata, :files => task.files}
+      {:metadata => task.metadata, :files => task.files, :module => pup_module.path}
```

Comment:
This may be clearer if it's module_dir I would expect module to be the module name.